### DISCUSSION TITLE
Is there a reason that both GitHub Actions CI and Travis CI run?

### DISCUSSION THREAD

**Discussion by ajsirota** [mu-editor/mu#1348] [Q&A] [answered]
I opened a PR #1347 and was wondering why I saw both a Travis status and a GitHub Actions status. 

Is there a reason for the redundancy?

_sorry if this comes off wrong, it's just a question I had._

**Comment by carlosperate** [comment, marked as answer]
Mostly because we haven't had the chance to move one of the current test environments from Travis to GitHub actions :)

Basically the Travis service allowance for open source projects changed and we started migrating to GitHub issues, there is still the Arm environment which we haven't ported yet and that is the main reason why Travis is till running.

**Reply by ajsirota** [reply]
Thanks!
